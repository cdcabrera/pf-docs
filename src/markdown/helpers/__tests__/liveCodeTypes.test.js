const { liveCodeTypes } = require('../liveCodeTypes');

describe('LiveCodeTypes', () => {
  it('should return a specific list', () => {
    expect(liveCodeTypes).toMatchSnapshot('list');
  });
});
