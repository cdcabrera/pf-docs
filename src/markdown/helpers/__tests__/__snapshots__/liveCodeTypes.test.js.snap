// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LiveCodeTypes should return a specific list: list 1`] = `
Array [
  "js",
  "ts",
  "html",
]
`;
